### ISSUE TITLE
Markdown links with hyphens (-) are turned into triple em-dashes 

### ISSUE THREAD

**Issue by bessey** [Cobertos/md2notion#13]
When I upload a document which contains a link with hyphens, e.g.:
```
- [G Suite User Configuration](https://wiki.blah.com/books/aws-access/page/g-suite-user-configuration)
```
The resulting link URL is:
```
https://wiki.blah.com/books/aws%E2%B8%BBaccess/page/g%E2%B8%BBsuite%E2%B8%BBuser%E2%B8%BBconfiguration
```
`%E2%B8%BB` == `⸻` the triple em-dash symbol. Its breaking all my wiki links :(

Confusingly, if right click the link in Notion and choose "copy link", I get the correct hyphenated version out instead of the em-dash one. But if I just click the link, no such luck.

FWIW If I upload the Markdown file directly through Notion's UI, the bug does not occur.

I have checked with `hexdump` that the Markdown file I am importing from truly contains an ASCII hyphen, and it does:
```
> hexdump -cb "./013-Single Sign-On--AWS Federated Access.md"
...
00005a0   s   s   /   p   a   g   e   /   a   w   s   -   c   o   n   s
00005a0 163 163 057 160 141 147 145 057 141 167 163 055 143 157 156 163
...
```

**Comment by Cobertos** [comment]
Really busy with a work project but I should be able to get to this in the next week and a half.

**Comment by Cobertos** [comment]
Looks like this got fixed in `notion-py` @ version `0.0.26`. So that just needs a version bump and it should be good.

**Comment by Cobertos** [comment]
Bumped the version, this should be fixed
